test/jest/components/PagesList.test.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import PagesList, { Page } from 'components/PagesList';
+import Consts from 'utils/consts';
+
+describe('(Component) PagesList', () => {
+    // @ts-ignore To make typescript happy
+    const noop = (..._: any[]) => {};
+    const defaultProps = {
+        onPageSelected: noop,
+        onPageRemoved: noop,
+        onPageReorder: noop,
+        pages: [],
+        selected: undefined,
+        isEditMode: false
+    };
+    const samplePages: Page[] = [
+        { id: 'abafar', name: 'Abafar', isDrillDown: false, tabs: [], widgets: [] },
+        { id: 'mustafar', name: 'Mustafar', isDrillDown: false, tabs: [], widgets: [] }
+    ];
+
+    it('should renders component', () => {
+        const wrapper = mount(
+            <PagesList
+                onPageReorder={defaultProps.onPageReorder}
+                onPageRemoved={defaultProps.onPageRemoved}
+                onPageSelected={defaultProps.onPageSelected}
+                isEditMode={defaultProps.isEditMode}
+                pages={defaultProps.pages}
+            />
+        );
+
+        expect(wrapper).toHaveLength(1);
+    });
+
+    it('should renders menu item as link', () => {
```

Comment:
Similar, minor typo, probably better would be:
```suggestion
    it('should render menu item as link', () => {
```